### PR TITLE
weather command patch, In the off chance the 'deg' is nil

### PR DESCRIPTION
**Pull request by IceDragon200**
^ Title

### CODE REVIEW

File: commands/weather.rb [hunk 1]
```diff
@@ -38,8 +39,12 @@
         main = data['main']
         wind = data['wind']
         sys = data['sys']
-        arr = ["N","NNE","NE","ENE","E","ESE", "SE", "SSE","S","SSW","SW","WSW","W","WNW","NW","NNW"]
-        wind_dir = arr[((wind['deg'] / 22.5) + 0.5) % arr.size]
+
+        wind_dir = if wind_deg = wind['deg']
```
Comment (archSeer) [inline commands/weather.rb]: wind_dir = wind['deg'] ? dirs[((wind_deg / 22.5) + 0.5) % dirs.size] : 'UNKNOWN'

Comment (IceDragon200) [inline commands/weather.rb]: That goes over the 80 characters, still want it?

Comment (archSeer) [inline commands/weather.rb]: At the very least get rid of `wind_deg`, and just use `wind['deg']`

Comment (archSeer) [inline commands/weather.rb]: minor though :3

**Pull request merged**